website/language.py

```diff
@@ -108,6 +108,11 @@
 
 </ul>
 '''
+REGISTRATION_EMBARGO_INFO = '''
```

Comment:
@bnosek @saradbowman This is the message users will be presented with while filling out the make public / enter into embargo section registration template page. Need assistance filling this in